README.md

```diff
@@ -303,6 +303,7 @@ Nick Le Large&nbsp;
 
 If you use arbitration graphs in your research, we would be pleased if you cite our work:
 
+{% raw %}
```

Comment:
I assume this is an adjustment for the pages? On the GitHub rendered readme this doesn't do anything and I'd assume that people will often see this one first before going to the website...

Reply:
Good catch, that's a no-go!

I assume'd the GitHub Readme rendering uses Jekyll as well. Probably it has a more restrictive setup. 

The issue on Pages is that our bibtex titles with double curly brackets are eaten by Liquid (Jekylls template language). 

`Decision-{{Making}}` is rendered `Decision-`

We should remove the brackets then.
BetterBibLaTeX set them to ensure capitalization is preserved right when cited. 🤷‍♂️